mlxtend/plotting/pca_correlation_graph.py

```diff
@@ -55,10 +75,9 @@ def plot_pca_correlation_graph(X, variables_names, dimensions=(1, 2),
     X = X - X.mean(axis=0)
     n_comp = max(dimensions)
 
-    if X_pca is None:
-        pca = PrincipalComponentAnalysis(n_components=n_comp)
-        pca.fit(X)
-        X_pca = pca.transform(X)
+    pca = PrincipalComponentAnalysis(n_components=n_comp)
```

Comment:
In the previous version, we allowed users to provide their own PC axes as input to the plot. In this updated version, we know have to run PCA on the dataset for producing the plot. I think this simplifies the use/API in general and would generally be ok, but I am curious: what would be the reason for this change?